src/core/lang.mli

```diff
@@ -220,10 +197,8 @@ val error_t : t
 val source_t : ?methods:bool -> t -> t
 val of_source_t : t -> t
 val format_t : t -> t
-val kind_t : Frame.kind -> t
-val kind_none_t : t
-val frame_t : t Frame.Fields.t -> t
-val of_frame_t : t -> t Frame.Fields.t
+val frame_t : t -> t Frame.Fields.t -> t
+val internal_t : unit -> t
```

Comment:
Add a comment here. Does this correspond to the default frame type with internal content types?